src/GRA.Web/Areas/MissionControl/Views/DailyTips/Detail.cshtml

```diff
@@ -0,0 +1,157 @@
+﻿@model GRA.Controllers.ViewModel.MissionControl.DailyTips.TipDetailViewModel
+@using GRA.Controllers.ViewModel.MissionControl.DailyTips
+
+<h2>Tip: @Model.Tip.Name</h2>
+<p>@Model.Tip.Message</p>
+
+<div class="row py-4">
+    <div class="col-12">
+        <a class="btn btn-outline-success"
+           asp-action="Add"
+           asp-route-tipId="@Model.Tip.Id">
+            <span class="far fa-file-archive fa-fw"></span>
+            Add new Daily Tips
+        </a>
+    </div>
+</div>
+
+<div class="row">
+    <div class="col-12">
+        <table class="table link-table table-bordered table-sm">
+            <thead>
+                <tr>
+                    <th>Day</th>
+                    <th>Name</th>
+                    <th>Extension</th>
+                    <th>Preview</th>
+                    <th>Actions</th>
+                </tr>
+            </thead>
+            <tbody>
+                @if (Model?.Images?.Any() != true)
+                {
+                    <tr>
+                        <td colspan="5">
+                            <div class="alert alert-warning mb-0">No images found.</div>
+                        </td>
+                    </tr>
+                }
+                else
+                {
+                    foreach (var image in Model.Images)
+                    {
+                            <tr>
+                                <td>@image.Day</td>
+                                <td>@image.Name</td>
+                                <td>@image.Extension</td>
+                                <td>
+                                    <img src="@Url.Content($"~/{image.ImagePath}")"
+                                         alt="@image.Name"
+                                        style="max-height: calc(50vh - 100px); overflow-y:auto;" />
+                                </td>
+                                <td class="btn-group ps-2" role="group">
+                                    <form asp-action="DeleteImage"
+                                          asp-route-imageId="@image.Id"
+                                          asp-route-tipId="@Model.Tip.Id"
+                                          method="post"
+                                          class="d-inline">
+                                        <button type="submit" class="btn btn-outline-danger btn-sm">
```

Comment:
If you add the class `button-spinner` to this and the following two buttons:
1. It should change to a spinner on click
2. It should automatically disable the button locking out double-clicks
3. You can call `ResetSpinners()` to reset all spinners on the page or `ResetSpinners($("#elementId"))` to reset the spinner on that particular element.

Once that's in place you can take out your `fa-spinner` class check and your `.addClass` and `.removeClass` calls related to that.